test/podFailover/controller/cmd/main.go

```diff
@@ -251,18 +265,18 @@ func getFailureType(pod v1.Pod) (string, error) {
 	return failureType, nil
 }
 
-func getCrdAndReportFailure(ctx context.Context, podName string, namespace string, failureType string, clientset *podfailure.PodFailoverClient) error {
-	podfailureobj, err := clientset.PodFailures(namespace).Get(ctx, podName, metav1.GetOptions{})
+func getCrdAndReportFailure(ctx context.Context, podName string, namespace string, failureType string, clientset *podfailure.Clientset) error {
+	podfailureobj, err := clientset.PodfailureV1beta1().PodFailures(namespace).Get(ctx, podName, metav1.GetOptions{})
 	if err != nil {
 		return fmt.Errorf("failed to get crd for pod %q: %v", podName, err)
 	}
 
-	updatedPodFailure := &podfailure.PodFailure{}
+	updatedPodFailure := &podfailurev1beta1.PodFailure{}
 	podfailureobj.DeepCopyInto(updatedPodFailure)
```

Comment:
You could simply use `DepyCopy`, e.g. `updatedPodFailure := podfailureobj.DeepCopy()`, and avoid the need for importing the `podfailurev1beta1` package. It would reduce the churn if/when we update the API version.